### PR TITLE
Eventdetails

### PR DESCRIPTION
**Pull request by JuanOfMany**

### CODE REVIEW

File: example.env [hunk 1]
```diff
@@ -2,8 +2,8 @@ PORT=5000
 PGUSER='ubuntu'
 PGPORT='5432'
 PGDATABASE='waggl'
-PGHOST='insertEC2here'
-PGPASSWORD='weshouldcomeupwithone'
+PGHOST='34.229.167.164'
+PGPASSWORD='password'
```
Comment (maria6417) [inline example.env]: might not want to hardcode this into the example file if this is the actual password ...?

File: server/index.js [hunk 2]
```diff
@@ -13,7 +13,9 @@ const io = require('socket.io')(server, {
     origin: '*',
   },
 });
+
 app.use(cors());
+app.use(express.json());
```
Comment (maria6417) [inline server/index.js]: looks like theres already an express.json on line 38!

File: example.env (hunk 1 above)
Comment (maria6417) [reply example.env]: i think we should share these configs in slack for security purposes!

**Review by maria6417 — COMMENTED** [review]
just one quick fix for the example.env file !

File: server/index.js (hunk 2 above)
Comment (JuanOfMany) [reply server/index.js]: Good catch!

File: example.env (hunk 1 above)
Comment (JuanOfMany) [reply example.env]: It's DEFINITELY not the real password ;), but the example.env is gone now, you're completely right.

**Review by maria6417 — APPROVED** [review]
LGTM!

**Pull request merged**